test/forms/inputs/radio-group.test.js

```diff
@@ -131,3 +142,58 @@ test('RadioGroup passes down props to children', () => {
     expect(el).toHaveAttribute('data-test', 'true')
   })
 })
+
+test('when no custom required indicator provided, do not show required indicator', () => {
```

Comment:
Can you prefix this test (and the following) with `RadioGroup` to match the rest of the descriptions in this file?